### PR TITLE
FigureCard and unit tests

### PR DESCRIPTION
**Pull request by jantocv**
## Description
`FigureCard` currently is a component inside `mx-merino` that could be used across many different repos. 
The purpose of this PR is to integrate `FigureCard` as a new component to use from `b2c-mapp-ui`. 

### Doubts: 
- Should be a MAJOR since we are adding new functionality that is not backwards-compatible comparing with other versions? 4.0.0 version should be created in this case.
- SetUP Webpack to serve static images from `docs/images` and be ablet o see `lanaLogo.svg` displayed in the FigureCard section of the web documentation.

## Checks
- [ ] Requires documentation update
- [x] Requires lib version update
- [x] Unit tested

## Versioning impact
- [x] **MAJOR** incompatible API changes.
- [ ] **MINOR** adds functionality in a backwards-compatible manner.
- [ ] **PATCH** backwards-compatible bug fixes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lana/b2c-mapp-ui",
-  "version": "3.2.2",
+  "version": "4.0.0",
```
Comment (rahulpsingh) [inline package.json]: We can keep this release just version `3.2.3`.

Comment (jantocv) [reply package.json]: Keeping 4.0.0... This is not a patch version that it implies a breaking change adding a component it was not present before.

**Comment by jantocv** [comment]
> Few more improvements,
> 
> * Please update the package version in `package-lock` as well.
> * Please update the changelog as well.

Done friend ;)

File: package.json (hunk 1 above)
Comment (rahulpsingh) [reply package.json]: I was just thinking it may create a bit of instability in versions.

**Pull request merged**